examples/natural_language_inference_deberta.rs

```diff
@@ -20,14 +20,13 @@ fn main() -> anyhow::Result<()> {
     let merges_resource = Box::new(RemoteResource::from_pretrained(
         DebertaMergesResources::DEBERTA_BASE_MNLI,
     ));
-    let model_resource = Box::new(RemoteResource::from_pretrained(
+    let mut model_resource = Box::new(RemoteResource::from_pretrained(
```

Comment:
note that with the latest changes (`RwLocks`) there is no need for the resource to be mutable anymore meaning the changes should be essentially backward compatible (examples/tests should be possible to leave unchanged)